core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -258,113 +258,155 @@ oppia.factory('changeListService', [
   }
 ]);
 
-oppia.factory('explorationPropertyService', [
-  '$rootScope', '$log', 'changeListService', 'AlertsService',
-  function($rootScope, $log, changeListService, AlertsService) {
-    // Public base API for data services corresponding to exploration properties
-    // (title, category, etc.)
-
-    var BACKEND_CONVERSIONS = {
-      param_changes: function(paramChanges) {
-        return paramChanges.map(function(paramChange) {
-          return paramChange.toBackendDict();
-        });
+// A data service that stores data about the rights for this exploration.
+oppia.factory('explorationRightsService', [
+  '$http', '$q', 'ExplorationDataService', 'AlertsService',
```

Comment:
@Dragneel7 You don't have to make any changes related to `ExplorationRightsService`. That's a separate part of the issue. Please undo all the changes involving this service and just keep the one for `ExplorationPropertyService`.

Reply:
Thanks for the review, I will do the required changes.